### ISSUE TITLE
iPad UI adjustments

### ISSUE THREAD

**Issue by samanpwbb** [mapbox/MapBoxiPad#101]
I am not suggesting any major adjustments at this point, just small adjustments to make the app feel more polished. I know that more radical UI reworking is on the horizon, but I do feel this intermediate step serves two purposes: first of all, it's my first shot at designing for the iPad, so I now have some groundwork laid to help start on a more involved redesign. Secondly, I'm suggest a few basic adjustments that should be easy to implement and also have immediate impact on the look and usability of the app. 

I'm going to post screenshots and then walk through the adjustments. 
## Home: https://img.skitch.com/20110624-t4yiddisyqg1k78e8rptre64n2.png
1. Menu bar is 50% transparent black over the map, and icons are fully opaque. While downloading and checking out apps over the last few days, I noticed this style in use on a few other maps. Aesthetically, it's the way to go for Mapbox for now.
2. I created icons items in the menu bar. The underlying active state color is #2c9eb8, which is related to the blue on the background of Mapbox.com. It's a tiny little branding detail that works nicely across the site. 

_Question: I did not make icons for "my maps" or "?". The "?" is already an icon basically, so that can stay. I played around with the idea of using an icon for "my maps", but ultimately decided against it. The best option if we did want to make icons here would be to create two: a “save” icon and a “load” icon, and flatten that hierarchy. I don't see a single icon working when there's the drop down. Justin/Will, what do you guys think?_
1. The layers pop-out is black. It matches the rest of the app, and due to the added transparenty on the toolbar, it’s visually separated enough that it doesn’t need it’s own hue. 
## Server Details: https://img.skitch.com/20110624-tq48t1yxp6qwfpdmxrd7u8ik1j.png

I changed the modal view style in one major way: the background is 70% transparent black. There’s also no line break between the navigation bar and content area. 
1. I dropped the traditional tabbed view in the current build of the app and instead went with Apple’s segmented control bar style to switch between views.
2. The status spinner thing is in the address bar, and a green checkmark shows up if a valid address is input (https://skitch.com/samanpwbb/fgqbw/screen-shot-2011-06-24-at-1.46.33-am). I was playing with moving the “connect to server” button to the body of this view, but I imagine the main user action here will be to press “enter”.
3. The recent server list could easily have a scroll bar added to it if there were more than the 5 or 6 server addresses, right? I’m wondering if anyone has a better idea for how this view should look.
## Browse Server view: https://skitch.com/samanpwbb/fgqnw/skitched-20110624-043329

First thing you’ll notice here is that I dropped cover flow. We might want to move away from cover flow for a couple reasons: firstly, the grid view more closely mirrors how the web UI of Tilestream looks, and if users are familiar with that, they’ll know what to expect here too. Grid view also allows you to have a better overview of which maps you’ve already selected.
1. Do you think this is text is instructional enough? We could go add “click the top right corner for a map preview” as well. 
2. Firstly, There’s a background highlight rather than a small icon when a layer is selected. Secondly, notice that the zoom layer info isn’t displayed on the main view. I am following the TileStream web UI’s footsteps with this choice, it reduces clutter and isn’t very essential information. Thirdly, Is there enough space for map titles here?
3. A click on the top right corner goes to the preview. The dogear+magnifying glass that could be toned down or we could cut it and revert back to a gesture-based preview.
4. horizontal pagination or vertical scrolling could work here, I went with pagination in the mock-up.
## Map preview: https://skitch.com/samanpwbb/fgqdg/map-preview

Only thing new here is the added metadata on the bottom.
## In conclusion,

I’d love to here feedback or counterpoints to some of the things I’m bringing up here if you have them. I’m still definitely not that comfortable designing for the iPad and there are probably a few things I’m not picking up on or things I did wrong.

Justin, what do you need from me in the way of graphical assets, like icons? 

Would more detailed explanations of metrics and styles be useful? These mock-ups are designed at 100% and are basically pixel perfect, with a few shortcuts taken here and there.
